core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -561,7 +553,9 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
         .last(newLastScanTime - 1)
         .asScala
         .toList
-      stale.foreach { log =>
+      stale.filterNot { log =>
+        processing.contains(log.logPath.split("/").last)
```

Comment:
isProcessing can be reused here and below ?

Reply:
Thanks.